README.md

```diff
@@ -2,254 +2,240 @@
+### Forking the Module 0 Capstone Repository
 
-Next, we are going to *fork this repository*. Forking is when you copy a Github repository to your Github account to make your own changes. Think of it like a fork in the road -- you're about to make changes that differ from the main path.
+Next, we are going to *fork this repository that you're reading right now*. Forking is when you copy a Github repository to your Github account to make your own changes. Think of it like a fork in the road -- you're about to make changes that differ from the main path.
 
-In this scenario, the Turing Github account owns the `module_0_capstone` repository. You do not have permission to change anything in this repository, so you need your own copy to work on. In order to fork the repository, follow these steps:
+In this scenario, the [Turing Github account](https://github.com/turingschool) owns this `module_0_capstone` repository. You do not have permission to change anything in this repository, so you need your own copy to work on. In order to fork the repository, follow these steps:
 
-1. Make sure you are logged in to GitHub (if you are not logged in, log in and come back to this page)
-1. Scroll to the top of *this* page
-1. Click on `Fork` in the upper right corner of the screen _this will take you to a new page_.
-_Now, you should be on your forked copy of this repository!_
+####  1. Make sure you are logged in to GitHub (if you are not logged in, log in and come back to this page)
 
-![Fork](https://i.imgur.com/7a8vnMJm.png)
+####  2. Scroll to the top of [*this* page that you're reading right now](https://github.com/turingschool-examples/module_0_capstone).
 
-The next thing you need to do is *clone your forked repository*. Cloning is when you copy a remote Github repository to your local computer:
+####  3. Click on `Fork` in the upper right corner of the screen and select your account as the destination
 
-1. In YOUR `module_0_capstone` repository that your just forked, click on `Clone or Download`
-_If you see `Clone with HTTPS` click on `Use SSH`_
+![click on 'fork' button, confirm fork to your account](/images/fork_01.jpg)
 
-![Clone](https://i.imgur.com/RHpul4Vm.png)
+####  4. On the new page, confirm that it says it's the "forked" copy of this repository, with mentions of your username in the URL and repository name.
 
-1. Click on the copy icon to copy the SSH link to your clipboard.
-1. Open your terminal
-1. Enter `cd`. This is a shortcut for `cd ~`, which is "change into the home directory"
-1. Enter `mkdir turing`
-1. Enter `cd turing`
-1. Enter `mkdir 1module`
-1. Enter `cd 1module`
-1. Enter `git clone <paste the copied ssh link>`.
-_You now have a copy of this repository on your local machine!_
-1. Enter `ls`. You should see a directory called `module_0_capstone`
-1. Enter `cd module_0_capstone`
-1. Enter `git status`.
+![confirm new repository belongs to _your_ github account](/images/fork_02.jpg)
+
+## Clone down this new repository
```

Comment:
Cloning and forking are something that we briefly go over in Mod 0, however I think that this is useful for review.